feature/wallet-info/data/src/main/java/com/appcoins/wallet/feature/walletInfo/data/wallet/db/entity/WalletInfoEntity.kt

```diff
@@ -23,4 +23,5 @@ data class WalletInfoEntity(
   val fiatCurrency: String?,
   val fiatSymbol: String?,
   val canTransfer: Boolean,
+  val email: String?
```

Comment:
We must increase the DB scheme version since there is this new email field. It's needed to keep compatibility without needing to clear user data from the device.
We need to:
 - increase the scheme version
 - create a custom migration to treat the scenario where there is already a DB created and now the app tries to read this email field